next.config.js

```diff
@@ -3,6 +3,7 @@ const nextConfig = {
   typescript: {
     ignoreBuildErrors: true,
   },
+  reactStrictMode: false,
```

Comment:
added this because i am not convinced it is a good idea to have different behaviour between dev and prod, but strict mode is meant to do some additional checks for us

what do you think?

Reply:
I agree for now - can always reinstate